pkg/customresources/certificatestores/certificatestores.go

```diff
@@ -14,72 +14,83 @@ limitations under the License.
 package certificatestores
 
 import (
+	"context"
 	"crypto/x509"
+	"os"
+	"strings"
+	"sync"
 
 	"github.com/deislabs/ratify/internal/constants"
+	ctxUtils "github.com/deislabs/ratify/internal/context"
+	"github.com/deislabs/ratify/pkg/utils"
+	vu "github.com/deislabs/ratify/pkg/verifier/utils"
 )
 
 // ActiveCertStores implements the CertStoreManager interface
 type ActiveCertStores struct {
-	// TODO: Implement concurrent safety using sync.Map
-	// The structure of the map is as follows:
-	// The first level maps from scope to certificate stores.
-	// The second level maps from certificate store name to certificates.
+	// scopedCertStores is mapping from cert store name to certificate list.
 	// The certificate store name is prefixed with the namespace.
 	// Example:
 	// {
-	//   "namespace1": {
-	//     "namespace1/store1": []*x509.Certificate,
-	//     "namespace1/store2": []*x509.Certificate
-	//   },
-	//   "namespace2": {
-	//     "namespace2/store1": []*x509.Certificate,
-	//     "namespace2/store2": []*x509.Certificate
-	//   }
+	//   "namespace1/store1": []*x509.Certificate,
+	//   "namespace2/store2": []*x509.Certificate
 	// }
-	// Note: Scope is utilized for organizing and isolating cert stores. In a Kubernetes (K8s) environment, the scope can be either a namespace or an empty string ("") for cluster-wide cert stores.
-	ScopedCertStores map[string]map[string][]*x509.Certificate
+	// Note: The namespace "default" is reserved for cluster-wide scenario.
```

Comment:
I think this is out of date? "default" is not reserved right? It's the `EmptyNamespace` instead?

Reply:
thanks for catching! I did update in my last batch but forgot to push it. Actually as we disuccsed last week, there should be no cluster-wide concept while users keep using certStore since certStore will always prefixed with the namespace (default if users don't provide it). And a cluster-wide verifier could always access certStores in all namespaces as v1.1 supports.